docs/blockchain-rpc/hedera-rpc.md

```diff
@@ -0,0 +1,302 @@
+
+
+ _Hedera has a separate open-source project implementing parts of the [Ethereum JSON-RPC standard](https://docs.hedera.com/hedera/core-concepts/smart-contracts/json-rpc-relay) which is not covered in this documentation._
+
+## Methods
+
+- [`hedera_signAndExecuteTransaction`](#hedera_signandexecutetransaction)
+- [`hedera_signTransaction`](#hedera_signtransaction)
+- [`hedera_executeTransaction`](#hedera_executetransaction)
+- [`hedera_signAndExecuteQuery`](#hedera_signandexecutequery)
+- [`hedera_signMessage`](#hedera_signmessage)
+- [`hedera_getNodeAddresses`](#hedera_getnodeaddresses)
+
+## hedera\_signAndExecuteTransaction
+
+The `hedera_signAndExecuteTransaction` method is a generic method for executing a transaction on the Hedera network. A dApp can begin by constructing a transaction with one of the Hedera SDKs or by constructing the raw protobuf messages and may select one or more consensus nodes that are authorized to execute the transaction.
+
+The dApp then constructs a list of valid transaction bytes that differ only in the node account id and serializes the list, for example by using the `toBytes()` method of an SDK. Finally, the dApp base64 encodes the resulting bytes. This final base64 encoded string of bytes is sent as a method param titled `transactionList` to the wallet.
+
+Wallets and SDKs must take special care to verify that each transaction in the list differs only in the node that is authorized to submit the transaction and does NOT differ in intent before submitting to an end user for approval and ultimately signing.
+
+
+### Parameters
+
+    1. `Object` - signAndExecuteTransaction parameters
+      1.1. `signerAccountId` : `String` - Hedera account id in the format `<network>:<shard>.<realm>.<num>-<optional-checksum>`
+      1.2. `transactionList` : `String` - base64 encoded string of TransactionList bytes
+
+### Returns
+
+    1. `Object` - Result of transaction submission to Hedera network
+      1.1. `nodeId` : `String` - The Hedera node the transaction was submitted to
+      1.2. `transactionHash` : `String` - The hash of the transaction
+      1.3. `transactionId` : `String` - Transaction ID, which includes the payer account id and the valid start timestamp
+
+### Error
+
+In certain conditions, the Hedera network with return a response that signifies a pre-processing validation error, for example, when the transaction has expired. In these cases, wallets will return an error with the following format:
+
+    1. `Object` - Result of transaction submission to Hedera network
+      1.1. `code` : 9000 - the reserved Wallet Connect error code for unknown or errors not related to the Wallet Connect protocol
+      1.2. `message` : `String` - A human readable string describing the nature of the failure
```

Comment:
Is this human-readable string totally arbitrary/free-form? Is a blank message ok, or is the [string that corresponds to the applicable enum code](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/response_code.proto#L32) preferable as a fallback?  I only mention it because familiarity with the error-code mapping might be a lot to ask of a dapp developer whose first interaction with the hashiverse is adding hedera wallet support to a dapp in "evm-rpc mode only" 😅

Reply:
great callout

In practice this will be similar to 
```javascript
transaction ${props.transactionId.toString()} failed precheck with status ${props.status.toString()};
```

This is from the javascript SDK here - https://github.com/hashgraph/hedera-sdk-js/blob/c78512b1d43eedf1d8bf2926a5b7ed3368fc39d1/src/PrecheckStatusError.js#L48C1-L49C1

So, the `props.status.toString()` piece should give the string that corresponds to error code - https://github.com/hashgraph/hedera-sdk-js/blob/c78512b1d43eedf1d8bf2926a5b7ed3368fc39d1/src/Status.js#L42

I'd probably recommend not setting hard requirements on the format of this string, though definitely open to that if that's helpful